source/_components/joaoapps_join.markdown

```diff
@@ -33,20 +39,41 @@ joaoapps_join:
     api_key: asd97823jb628a34fwsdfwefd5384345tf2d
 ```
 
-Configuration variables:
+{% configuration %}
+api_key:
+  description: The API key for Join.
+  required: true
+  type: string
+device_id:
+  description: The id of your device.
+  required: false
+  type: string
+device_ids:
+  description: Comma separated list of device ids.
+  required: false
+  type: list
+device_names:
+  description: Comma separated list of device names.
+  required: false
+  type: list
```

Comment:
Comma-separated list: This is a `string`.

https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/joaoapps_join.py#L28